scripts/validate_docstrings.py

```diff
@@ -145,10 +145,17 @@ def examples_errors(self):
         runner = doctest.DocTestRunner(optionflags=flags)
         context = {"np": numpy, "pd": pandas}
         error_msgs = ""
-        for test in finder.find(self.raw_doc, self.name, globs=context):
-            f = io.StringIO()
-            runner.run(test, out=f.write)
-            error_msgs += f.getvalue()
+        name = None
+        try:
+            name = self.name
+        except AttributeError:
+            if not isinstance(self.obj, property):
+                name = type(self.obj).__name__
+        if name is not None:
```

Comment:
likewise, none of the tests cover when `name` is `None` - when could that happen?